test/smoke/framework/testCore/src/main/java/com/microsoft/applicationinsights/smoketest/AiSmokeTest.java

```diff
@@ -225,19 +234,38 @@ public void run() {
 	}
 
 	@ClassRule
-	public static TestWatcher agentModeLoader = new TestWatcher() {
+	public static TestWatcher classOptionsLoader = new TestWatcher() {
 		@Override
 		protected void starting(Description description) {
+			System.out.println("Configuring test class...");
 			UseAgent ua = description.getAnnotation(UseAgent.class);
```

Comment:
Please refrain from using the variable names like "ua". Understandable that it is test code, but more of it is a framework where people will contribute and it makes harder to read code and contribute.

Reply:
ok. This was a nitpick :) My only point was to ensure consistency. If this is a practice which is followed through out the smoketest framework, then I would suggest that you outline it in contributing.md so that there is a better chance of consistency when others contribute to it.